### ISSUE TITLE
Missing phaser_exe command line argument

### ISSUE THREAD

**Issue by fsimkovic** [rigdenlab/ample#7]
We currently cannot provide a different phaser executable to AMPLE. Is this intended behaviour @linucks ?

**Comment by linucks** [comment]
I'm not sure what you mean by intended behaviour @fsimkovic - it's just something we've not had a need for. It certainly doesn't seem like a bad idea to add it - although the default should obviously be to use the standard ccp4 binary

**Comment by fsimkovic** [comment]
@linucks definitely, I just realised it a while back when I wanted to run with an older version of Phaser that I had outside of the CCP4 distro.

To do this right now we only have 2 options, one is to replace the old binary with the new, or hard code it in the module ...

We provide flags for almost every binary, might as well do it for Phaser as well.
